figures/horizontal.py

```diff
@@ -0,0 +1,31 @@
+from cmath import pi
```

Comment:
I don't think we should include this file as it is a copy of `line.py` just with another name